charts/posthog/templates/web-deployment.yaml

```
@@ -63,6 +63,8 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         ports:
         - containerPort: {{ .Values.service.internalPort }}
+        # Expose the poort on which Prometheus /metrics endpoint resides
```

```suggestion
        # Expose the port on which Prometheus /metrics endpoint resides
```

lol copy past huh? ;)